content/docs/user-guide/data-pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,267 @@
+# Defining Pipelines
```

Comment:
Intro is too long, it goes into some abstract / advanced things before you see anything at all that looks like DVC pipeline or stage commands. I think we should restructure this- bring more `dvc.yaml` , stage add to the top

Reply:
Done. I also removed some details that relate more to reproduction (which will be a separate page).